src/dragging.js

```diff
@@ -27,12 +14,11 @@ function dragger(arr) {
   const item2 = arr[droppedItem];
   arr.splice(item1.index, 1);
   arr.splice(item2.index, 0, item1);
-  for (let i = 0; i < arr.length; i += 1) {
-    arr[i].index = i;
+  for (let i = 1; i <= arr.length; i += 1) {
+    arr[i - 1].index = i;
   }
-  printList(arr);
 }
 
 export {
-  dragger, heldItem, droppedOn, printList,
+  dragger, heldItem, droppedOn,
```

Comment:
[Optional]

- [ ] Drag and drop functionality doesn't work. **This should have been implemented in the previous milestone**. Please be sure this feature working.


https://user-images.githubusercontent.com/1642014/126027854-20d98670-fee8-48ed-891c-91ff2fa16171.mp4

Reply:
It was implemented originally but broke when I made changes to the project. I'll fix it before submitting a new request. Thank you.

Reply:
Fixed by changing the array index input on the dragger function. Thank you for bringing it up.